isites_migration/views.py

```diff
@@ -6,7 +6,6 @@
 
 from lti_school_permissions.decorators import lti_permission_required
 from async.models import Process
```

Comment:
Sure -- was just trying to minimize the refactoring, but happy to reorganize here..